core/matrix/ell.cpp

```diff
@@ -251,6 +256,35 @@ Ell<ValueType, IndexType>::extract_diagonal() const
 }
 
 
+template <typename ValueType, typename IndexType>
+void Ell<ValueType, IndexType>::compute_absolute_inplace()
+{
+    auto exec = this->get_executor();
+
+    exec->run(ell::make_inplace_absolute_array(
+        this->get_values(), this->get_num_stored_elements()));
+}
+
+
+template <typename ValueType, typename IndexType>
+std::unique_ptr<typename Ell<ValueType, IndexType>::outplace_absolute_type>
+Ell<ValueType, IndexType>::compute_absolute() const
+{
+    auto exec = this->get_executor();
+
+    auto abs_ell = outplace_absolute_type::create(
+        exec, this->get_size(), this->get_num_stored_elements_per_row(),
```

Comment:
Maybe also reset the stride like in Dense? Or do we use it for alignment purposes?

Reply:
Dense can be submatrix, so I ignore the stride.
In other format including ELL, I keep all information